src/Turing.jl

```diff
@@ -100,8 +100,8 @@ export @model,                 # modelling
     vi,                     # variational inference
     ADVI,
     sample,                 # inference
-    @logprob_str,
-    @prob_str,
+    @logprob_str,  # TODO: Remove, see https://github.com/TuringLang/DynamicPPL.jl/issues/356
```

Comment:
maybe we can setup a `todo` workflow like https://github.com/marketplace/actions/track-todo-action? (I haven't used it, just a thought)

Reply:
That sounds useful! We definitely have a lot of todos. Many of them labelled with names too. I'll open an issue for it